Makefile.build

```diff
@@ -454,7 +454,6 @@ check: validate test-suite
 test-suite: world $(ALLSTDLIB).v
 	$(MAKE) $(MAKE_TSOPTS) clean
 	$(MAKE) $(MAKE_TSOPTS) all
-	$(MAKE) $(MAKE_TSOPTS) report
```

Comment:
See 38d3aa9: test suite make all includes make report so it was printing the report twice.
For instance if we pick some random old build https://travis-ci.org/coq/coq/jobs/216784644:
>make[3]: Entering directory `/home/travis/build/coq/coq/test-suite`
BUILDING SUMMARY FILE
NO FAILURES
make[3]: Leaving directory `/home/travis/build/coq/coq/test-suite`
make[2]: Leaving directory `/home/travis/build/coq/coq/test-suite`
make -C test-suite -s VERBOSE= report
make[2]: Entering directory `/home/travis/build/coq/coq/test-suite`
BUILDING SUMMARY FILE
NO FAILURES
make[2]: Leaving directory `/home/travis/build/coq/coq/test-suite`